15/umbraco-cms/reference/routing/custom-routes.md

```diff
@@ -120,7 +120,7 @@ public IActionResult Product(string id)
 }
 ```
 
-This method is a bit more interesting, here we get some extra data from a different source, in this case a `DbContext`, but this can be anything you want, using the id we get from the route values. We use this extra data to create a custom model, wich includes the available stores, which we then render the view with.
+This method is a bit more interesting, here we get some extra data from a different source, in this case a `DbContext`, but this can be anything you want, using the id we get from the route values. We use this extra data to create a custom model, which includes the available stores, which we then render the view with.
```

Comment:
```suggestion
Here, we get some extra data from a different source. In this case, a `DbContext`, but this can be anything you want, using the ID we get from the route values. We use this extra data to create a custom model, which includes the available stores, which we then render the view with.
```